.github/workflows/webext-lint.yml

```
@@ -0,0 +1,22 @@
+name: Web-Ext lint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

For consistency with the suggested changes in #135:
```suggestion
      - uses: actions/checkout@main
```